lightsoff_api/__init__.py

```diff
@@ -40,6 +40,7 @@ class Place(db.Model):
     report_count = sa.Column(sa.Integer, nullable=False, default=1)
     latitude = sa.Column(sa.Float, nullable=False)
     longitude = sa.Column(sa.Float, nullable=False)
+    contacted = sa.Column(sa.Integer, default=0)
```

Comment:
A boolean would be more explicit ;)
```suggestion
    contacted = sa.Column(sa.Boolean, default=False)
```

Reply:
Oh my bad, that's a counter.

Maybe name it `contacted_count/times`, your call